src/frontend/package.json

```diff
@@ -55,6 +55,8 @@
     "@types/react-autosuggest": "10.0.0",
     "@types/react-dom": "16.9.8",
     "@types/react-modal": "3.10.6",
+    "@typescript-eslint/eslint-plugin": "^4.1.0",
```

Comment:
Dependencies are always pin in our project. You must set a specific version. We have a bot dependency responsible to update them.